qa/tests/build-jts.xml

```diff
@@ -83,7 +83,7 @@
         <then>
             <property name="jts.idl.tests.jar" value="${jts.build.dir}/idl/${orbtype}/target/jacorb-idl-${narayana.version}-tests.jar"/>
 		    <property name="org.jboss.jbossts.qa.orb.home" value="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/jacorb"/>
-		    <property name="org.jboss.jbossts.qa.orb.classpath" value="${org.jboss.jbossts.qa.orb.home}/lib/idl.jar;${org.jboss.jbossts.qa.orb.home}/lib/jacorb.jar;${org.jboss.jbossts.qa.orb.home}/lib/logkit-1.2.jar"/>
+		    <property name="org.jboss.jbossts.qa.orb.classpath" value="${org.jboss.jbossts.qa.orb.home}/lib/idl.jar;../ext/jacorb-omgapi.jar;${org.jboss.jbossts.qa.orb.home}/lib/logkit-1.2.jar"/>
```

Comment:
@mmusgrov FYI given a recent change (https://github.com/jbosstm/narayana/commit/00cdc7aa268e8ce2008a33ca414bce6304d2de08) QA_JTS_JACORB would have started using the version of JacORB which is checked in (a Jar that comes from [ext/](https://github.com/jbosstm/narayana/tree/master/ArjunaJTS/jacorb/lib) in the build system). I think before either that change or this PR the build system might have been using OpenJDK ORB rather than JacORB for the OMG API which isn't right for JacORB either way. Anyway after merging this PR we will now be using JacORB 3.7 here - given https://github.com/jbosstm/narayana/blob/master/qa/TaskImpl.properties.template#L39 I think the running (not compiling) of the tests might have already been using the OMG API from JacORB 3.7 (not the 3.7 implementation though). I am keeping the change to suppy the 3.7 API to the compiler and merging this PR - if you would like to see a further change here please let me know.